viewing_party/party.py

```diff
@@ -1,23 +1,143 @@
+import copy
+
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+
+    if not title or not genre or not rating:
+        return None

```

Comment:
It would be a bit more efficient to do the check if check before creating the dictionary. That way if any of the values don't make sense we don't need to take the time/space to create the dictionary.